docs/source/reference/tpu.rst

```diff
@@ -112,15 +190,18 @@ Now, we show a complete YAML for running `MNIST training <https://cloud.google.c
 
    TPU node requires loading data from a GCS bucket, so we add a :code:`file_mounts` to create a new bucket.
```

Comment:
1. We mount it to /dataset, but it's not used as an argument to mnist_main.py. Expected?
2. It's surprising to see L175-177 being required since we already mount to /dataset? Expected?

Reply:
Ah actually, mounting locally is not necessary here. TPU device will directly read/write data to the bucket (that's why we need to pass L175-177).
In fact, all I need is just creating a bucket. However, it's neither MOUNT nor COPY so I didn't know which to choose.

Reply:
But I think mounting could still be useful if user want to do some analysis on the CPU host VM.